ci/terraform/oidc/ecc-signing-key.tf

```diff
@@ -64,8 +64,10 @@ data "aws_iam_policy_document" "storage_token_signing_key_access_policy" {
   }
 }
 
+//NOTE: This key is used to sign the storage token included as a claim inside the
```

Comment:
I would try and get the information in this note into the `description` below

Reply:
Will update description, re-naming should help make it clearer